arch/arm/src/sama5/sam_ohci.c

```diff
@@ -3899,20 +3907,18 @@ static void sam_disconnect(struct usbhost_driver_s *drvr,
   DEBUGASSERT(rhport != NULL && hport != NULL && hport->ep0);
   ep0 = (struct sam_eplist_s *)hport->ep0;
 
-  /* Remove the disconnected port from the control list */
-
-  sam_ep0dequeue(ep0);
```

Comment:
Calling this when ep0 is NULL will cause a crash.
Should only be called for a root hub.